Dockerfile.nginx-alpine

```diff
@@ -164,5 +164,5 @@ ENV \
   METRICS_HTTP_AUTH_USER="" \
   METRICS_HTTP_AUTH_PASSWORD=""
 USER ${uid}
-EXPOSE 8080
+
```

Comment:
Please also add in Dockerfile.nginx-debian

Reply:
Resolved